src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -25,6 +25,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="Helpers.cs" />
+    <Compile Include="Microsoft\Win32\SafeHandles\SafeWaitHandle.cs" />
```

Comment:
I just moved these tests from System.Runtime.Handles.Tests to System.Runtime.Tests since the APIs they're testing moved. I'm not sure how the tests get compiled for an earlier framework version, but I was thinking since the tests don't reference any new APIs, they should continue to work. I figured the only difference is that the test is in a different test assembly. For that to work, would I need to add a reference to System.Runtime.Handles, since that's where the APIs used to be in the earlier framework version?